ACPs/181-p-chain-epoched-views/README.md

```diff
@@ -101,7 +101,7 @@ type Block interface {
 }
 
 func GetPChainEpoch(parent Block) Epoch {
```

Comment:
Perhaps out of scope of this fix, But I think it would be cleaner if we did:
```go
func GetPChainEpoch(parent Block) Epoch {
	parentTimestamp := parent.Timestamp()
	parentEpoch := parent.Epoch()
	epochEndTime := parentEpoch.StartTime.Add(D)
	if parentTimestamp.Before(epochEndTime) {
		// If the parent was issued before the end of its epoch, then it did not
		// seal the epoch.
		return parentEpoch
	}

	// The parent sealed the epoch, so the child is the first block of the new
	// epoch.
	return Epoch{
		PChainHeight: parent.PChainHeight(),
		Number:       parentEpoch.Number + 1,
		StartTime:    parentTimestamp,
	}
}
```

Essentially, I think `Before` is easier to read/understand than `not Before`. Also I just find the comments overly verbose... It just says what the code does.